gnome-extra/chrome-gnome-shell/chrome-gnome-shell-7.ebuild

```diff
@@ -19,15 +19,26 @@ IUSE=""
 
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
+DEPEND="${PYTHON_DEPS}
+	sys-apps/coreutils
+"
 RDEPEND="${PYTHON_DEPS}
 	dev-python/pygobject:3[${PYTHON_USEDEP}]
 	gnome-base/gnome-shell
 "
 
 src_configure() {
 	python_setup
-	python_fix_shebang ./connector
 
 	local mycmakeargs=( -DBUILD_EXTENSION=OFF )
 	cmake-utils_src_configure
 }
+
+src_install() {
+	echo -n "CONFIG_PROTECT_MASK=\"" > 50"${PN}"
+	echo -n "/etc/chromium/policies/managed/${PN}.json " >> 50"${PN}"
+	echo "/etc/opt/chrome/policies/managed/${PN}.json\"" >> 50"${PN}"
```

Comment:
as soon as you do filesystem I/O with external commands, you need `|| die` on them